core/gradient.js

```diff
@@ -23,7 +24,21 @@ GradientPrototype.add = function(stop) {
 }
 
 GradientPrototype.toString = function() {
-	return 'linear-gradient(' + this.orientation + ',' + this.stops.join() + ')'
+	if (this.type === $core.Gradient.Linear) {
+		return 'linear-gradient(' + this.orientation + ',' + this.stops.join() + ')'
+	}
+	else if (this.type === $core.Gradient.Conical) {
+		let gradientString = 'conic-gradient(from ' + this.orientation
+		for(let i = 0; i < this.stops.length; ++i) {
+			let stop = this.stops[i]
+			gradientString += ', ' + stop.color.hex() + ' '  + 2 * Math.PI * stop.position + 'rad'
+		}
+		gradientString += ')'
+		return gradientString;
+	}
+
+	console.error("Type is not supported for gradient:", this.type);
```

Comment:
```suggestion
	throw new Error("Gradient Type " + this.type + " is not supported")
```

and remove return please

Reply:
Done